Lib/test/test_asyncio/test_tasks.py

```diff
@@ -1293,32 +1320,6 @@ async def foo():
         self.assertAlmostEqual(0.15, loop.time())
         self.assertEqual(res, 42)
 
-    def test_wait_with_global_loop(self):
```

Comment:
It is virtually a duplicate of the above test. Initially they were different by passing the loop argument or not, now it is removed